data/schema/units/abilities.cfg

```diff
@@ -40,6 +40,11 @@
 		[/key]
 		{FILTER_TAG "filter" unit ()}
 	[/tag]
+	[tag]
+		name="affect_distant"
+		{REQUIRED_KEY radius s_int}
```

Comment:
Is there technical reason why it has to be radius-based instead of supporting arbitrary location filter? Filtering may be tricky, but I believe there are formula variables that provide access to ability owner location.